src/extension/content-script/onendnostr.js

```diff
@@ -43,14 +43,16 @@ async function init() {
     if (ev.data && !ev.data.response) {
       // if an enable call railed we ignore the request to prevent spamming the user with prompts
       if (isRejected) {
-        console.error(
-          "Enable had failed. Rejecting further WebLN calls until the next reload"
-        );
+        postMessage(ev, {
+          error:
+            "window.nostr call cancelled (rejecting further window.nostr calls until the next reload)",
+        });
         return;
       }
+
       // if a call is active we ignore the request
       if (callActive) {
-        console.error("nostr call already executing");
+        postMessage(ev, { error: "window.nostr call already executing" });
```

Comment:
is callActive still needed now that we support calls executing in parallel?

Reply:
we don't support sending in parallel. there is still only one request at a time. i think the main purpose of this if statement is to prevent websites from being able to spin up a larger amount of payment windows.